packages/curve-ui-kit/src/themes/typography.ts

```diff
@@ -89,13 +89,13 @@ export const TYPOGRAPHY_VARIANTS = {
   buttonS: { fontFamily: 'Button', fontWeight: 'Bold', fontSize: 'sm', lineHeight: 'md', textCase: 'uppercase' },
   buttonM: { fontFamily: 'Button', fontWeight: 'Bold', fontSize: 'md', lineHeight: 'xl', textCase: 'uppercase' },
 
-  tableHeaderM: { fontFamily: 'Body', fontSize: 'md', lineHeight: 'sm' },
-  tableHeaderS: { fontFamily: 'Body', fontSize: 'sm', lineHeight: 'xs' },
-  tableCellL: { fontFamily: 'Heading', fontWeight: 'Bold', fontSize: Sizing[250], lineHeight: Sizing[250] },
-  tableCellMRegular: { fontFamily: 'Mono', fontSize: 'md', lineHeight: 'sm' },
-  tableCellMBold: { fontFamily: 'Mono', fontWeight: 'Bold', fontSize: 'md', lineHeight: 'sm' },
-  tableCellSRegular: { fontFamily: 'Mono', fontSize: 'sm', lineHeight: 'xs' },
-  tableCellSBold: { fontFamily: 'Mono', fontWeight: 'Bold', fontSize: 'sm', lineHeight: 'xs' },
+  tableHeaderM: { fontFamily: 'Body', fontWeight: 'Medium', fontSize: 'md', lineHeight: 'sm' },
```

Comment:
This change affects the llamelend data table as well. Prob discussed this with tutti?